app/src/main/java/ar/com/wolox/android/login/ui/training/TrainingFragment.java

```diff
@@ -0,0 +1,68 @@
+package ar.com.wolox.android.login.ui.training;
+
+import android.content.Intent;
+import android.os.Bundle;
+import android.support.annotation.NonNull;
+import android.support.annotation.Nullable;
+import android.util.Log;
+import android.view.View;
+import android.widget.TextView;
+
+import ar.com.wolox.android.R;
+import ar.com.wolox.android.login.ui.viewpager.ViewpagerActivity;
+import ar.com.wolox.wolmo.core.fragment.WolmoFragment;
+
+import javax.inject.Inject;
+
+import butterknife.BindView;
+import butterknife.OnClick;
+
+public class TrainingFragment extends WolmoFragment<LoginPresenter> implements ILoginView {
+
+    @BindView(R.id.fragment_login_email) TextView mEmail;
+    @BindView(R.id.fragment_login_password) TextView mPassword;
+
+    @Inject
+    public TrainingFragment() {}
+
+    @Override
+    public int layout() {
+        return R.layout.fragment_login;
+    }
+
+    @Override
+    public void init() {
+        mEmail.setText(getPresenter().getEmail());
+        mPassword.setText(getPresenter().getPassword());
+    }
+
+    @Override
+    public void onCreate(@Nullable Bundle savedInstanceState) {
```

Comment:
Please remove empty overrides.
If you will only call `super.onCreate(savedInstanceState)` you can remove this `override`.